yarn.lock

```diff
@@ -1891,6 +1891,45 @@
     call-me-maybe "^1.0.1"
     glob-to-regexp "^0.3.0"
 
+"@neo-one/cli@1.1.1":
```

Comment:
One of the package.json dependencies is wrong, can you figure out why this is getting pulled in?